thoth/storages/si_bandit.py

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env python3
+# thoth-storages
+# Copyright(C) 2020 Kevin Postlethwait
+#
+# This program is free software: you can redistribute it and / or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+"""Adapter for storing bandit security indicators (SI) onto a persistence remote store."""
+
+from .result_base import ResultStorageBase
+
+
+class SecurityIndicatorResultsStore(ResultStorageBase):
```

Comment:
class names should be changed as well :)

Reply:
And please [add them also to the `__init__.py` file](https://github.com/thoth-station/storages/blob/master/thoth/storages/__init__.py) so we can easily import them.